crates/ruff_cli/Cargo.toml

```diff
@@ -64,6 +64,8 @@ wild = { version = "2" }
 
 [dev-dependencies]
 assert_cmd = { version = "2.0.8" }
+insta = { workspace = true, features = ["filters"] }
```

Comment:
Do we need to specify features if they're specified at the workspace level? (I genuinely don't know.)

Reply:
surprisingly, yes